e2e-tests/tfm/cypress/e2e/journeys/case/parties/bond-issuer.spec.js

```diff
@@ -208,6 +208,29 @@ context('Bond issuer URN - User can add, edit, confirm and submit URN to the TFM
 
         pages.partiesPage.bondIssuerEditLink().should('exist');
       });
+
+      it('should not remove the exporter party URN when the issuer URNs are submitted', () => {
+        const exporterUrn = CONSTANTS.PARTY_URN.ANOTHER_VALID;
+        submitExporterUrn(exporterUrn);
+        pages.partiesPage.bondIssuerEditLink().click();
+        pages.bondIssuerPage.urnInput(1).clear();
+        pages.bondIssuerPage.urnInput(2).clear();
+        pages.bondIssuerPage.urnInput(1).type(partyUrn[0]);
+        pages.bondIssuerPage.urnInput(2).type(partyUrn[1]);
+
+        pages.bondIssuerPage.saveButton().click();
+        pages.bondIssuerPage.saveButton().click();
+
+        pages.partiesPage.exporterUrn().should('contain', exporterUrn);
+      });
+
+      function submitExporterUrn(urn) {
+        pages.partiesPage.exporterEditLink().click();
+        pages.exporterPage.urnInput().clear();
+        pages.exporterPage.urnInput().type(urn);
+        pages.exporterPage.saveButton().click();
+        pages.exporterPage.saveButton().click();
```

Comment:
Why twice?

Reply:
The first time you click save takes you to a summary page that asks you to confirm your input. The second time you click save does the actual saving